addons/web/static/src/legacy/scss/bootstrap_overridden_frontend.scss

```diff
@@ -22,3 +22,7 @@ $link-decoration: none !default;
 // a muted one. This is defined here to be consistent in the whole frontend, as
 // when used with background classes, it should not be forced to a gray color.
 $figure-caption-color: inherit !default;
+
+// HR
+$hr-color: rgba(0, 0, 0, 0.1) !default;
+$hr-opacity: 1 !default;
```

Comment:
Reading this, I wondered why BS5 introduced `$hr-opacity` if you can simply put it inside the `$hr-color` variable as you did. It turns out it make sense to not have `$hr-opacity` set to `1`, for two reasons:
- Normal `<hr>` could follow the color of their surrounding text automatically but bad faded a bit
- They introduced vertical rulers thanks to the `vr` class, which does the same: follows the color of the text but faded thanks to the opacity a bit.

Setting this variable to 1 prevents us to use that class in the future. And in the end, the new BS5 behavior regarding `<hr>` is better, we should not break it for the sake of compatibility.

In this case, I would:
- Remove the customization of those variables, let them have their default BS5 values. Yes, default `<hr>` in pages could look a bit different (faded surrounding-text color instead of light gray), but it's not worth having compatibility code for that IMO. BS5 implies way more "visually breaking" changes than that. And in the end, we would prevent new 16.0 users to use normal BS5 hr, which are nicer than BS4 ones :grimacing: 
- Set opacity to 1 on the hr of s_hr snippet, for compatibility but mainly as I guess that in that case, we don't want the nice BS5 auto-behavior but rather want people to be able to set the color to the exact color they want to. Maybe the option should be smarter like "if there is no set color by the user, keep the auto faded text color"... but let's forget about that for now. Maybe add a comment next to the `opacity: 1` rule explaining that.